CITATION.cff

```diff
@@ -0,0 +1,59 @@
+cff-version: 1.2.0
+title: caput
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+doi: 10.5281/zenodo.5846374
+repository-code: 'https://github.com/radiocosmology/caput'
+abstract: >-
+  Useful utilities for dealing with large datasets on computer clusters with
+  applications to radio astronomy in mind. Includes modules for dynamically importing
+  and utilizing mpi4py, in-memory mock-ups of h5py objects, and infrastructure for
+  running data analysis pipelines on computer clusters.
+license: MIT
+authors:
+- given-names: "J. Richard"
+  family-names: "Shaw"
+  affiliation: "University of British Columbia"
+  orcid: "https://orcid.org/0000-0002-4543-4588"
+
+- given-names: "Kiyoshi"
+  family-names: "Masui"
+  affiliation: "MIT"
+  orcid: "https://orcid.org/0000-0002-4279-6946"
+
+- given-names: "Rick"
+  family-names: "Nitsche"
+  affiliation: "University of British Columbia"
+
+- given-names: "Anja"
+  family-names: "Boskovic"
+  affiliation: "University of British Columbia"
+
+- given-names: "Shifan"
+  family-names: "Zuo"
+
+- given-names: "Tristan"
+  family-names: "Pinsonneault-Marotte"
+  affiliation: "University of British Columbia"
+  orcid: "https://orcid.org/0000-0002-9516-3245"
+
+- given-names: "Liam"
+  family-names: "Gray"
+  affiliation: "University of British Columbia"
+
```

Comment:
orcid: "https://orcid.org/0000-0003-3986-954X"

Apologies, totally forgot about this. Looks good otherwise